pkg/runner/local_docker.go

```diff
@@ -225,6 +215,7 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow *rp
 
 		// Serialize the runenv into env variables to pass to docker.
 		env := conv.ToOptionsSlice(runenv.ToEnvVars())
+		env = append(env, "INFLUXDB_ADDR=http://testground-influxdb:8086")
```

Comment:
Not sure why we have both `INFLUXDB_HOST` and `INFLUXDB_ADDR`... ?

Reply:
`INFLUXDB_HOST` is the hostname for the sidecar.
`INFLUXDB_ADDR` is the http endpoint that the API client needs.

Reply:
You can infer one from the other, and it'd be good if we reduce to the minimum service discovery parameters necessary to spin up the system, and keep just `INFLUXDB_HOST`. I believe we do the same for REDIS, but we don't send `http://redis:6379` as an env var.

Reply:
That would require hardcoding the port number in the SDK. The `INFLUXDB_HOST` doesn't carry the port number. I don't really like doing that (even though we are doing it for Redis, but that's a vestige more than a desire).

Reply:
Yeah, that's what I meant - these dependencies are really the same (albeit one required and the other optional), so the service discovery can work in absolutely the same way. Redis is on 6379, Influx is on 8086 - I don't see anything bad with that.